### PR TITLE
Refine Keras Examples for INC New API

### PR DESCRIPTION
**Pull request by zehao-intel**
## Type of Change

feature or bug fix or documentation or validation or others  
API changed or not: No

## Description

detail description 
JIRA ticket: https://jira.devtools.intel.com/browse/ILITV-2451

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR : refine examples

## How has this PR been tested?

how to reproduce the test (including hardware information): extension

## Dependency Change?

any library dependency introduced or removed: No

### CODE REVIEW

**Comment by zehao-intel** [comment]
Extension: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3604/

File: examples/tensorflow/image_recognition/keras_models/resnetv2_101/quantization/ptq/main.py [hunk 1]
```diff
@@ -42,15 +39,14 @@
...
-flags.DEFINE_string(
-    'config', 'bert.yaml', 'yaml configuration of the model')
-
 flags.DEFINE_string(
     'calib_data', None, 'location of calibration dataset')
 
 flags.DEFINE_string(
     'eval_data', None, 'location of evaluate dataset')
 
+flags.DEFINE_integer('batch_size', 1, 'batch_size')
```
Comment (lvliang-intel) [inline examples/tensorflow/image_recognition/keras_models/resnetv2_101/quantization/ptq/main.py:48]: The default value of 'batch_size' should be set to 32.

File: examples/tensorflow/image_recognition/keras_models/resnetv2_101/quantization/ptq/main.py [hunk 2]
```diff
@@ -59,70 +55,73 @@
...
+        q_model = quantization.fit(FLAGS.input_model, conf=conf, calib_dataloader=calib_dataloader,
+                    eval_func=evaluate)
+        q_model.save(FLAGS.output_model)
+
+    if FLAGS.benchmark:
+        from neural_compressor import benchmark
+        from neural_compressor.config import BenchmarkConfig
+        if FLAGS.mode == 'performance':
+            conf = BenchmarkConfig(iteration=100, cores_per_instance=4, num_of_instance=7)
+            benchmark.fit(FLAGS.input_model, conf, b_dataloader=eval_dataloader, b_func=evaluate)
```
Comment (lvliang-intel) [inline examples/tensorflow/image_recognition/keras_models/resnetv2_101/quantization/ptq/main.py:118]: b_func should be removed, otherwise the iteration will not be controlled by INC inside.

**Comment by chensuyue** [comment]
extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3633/artifact/report.html

**Comment by zehao-intel** [comment]
Rebuild extension: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3643

**Comment by ftian1** [comment]
@chensuyue  this PR is modifying old examples to use new API. is it recommended way to do that? shall we add totally new examples?

**Review by ftian1 — APPROVED** [review]

**Comment by chensuyue** [comment]
> @chensuyue this PR is modifying old examples to use new API. is it recommended way to do that? shall we add totally new examples?

As we synced before all the old examples should be modified into new API or deleted in v2.0.

**Comment by zehao-intel** [comment]
Extension: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3649/

**Comment by ftian1** [comment]
looks like extenstion tests all fail. pls check why

**Comment by ftian1** [comment]
> As we synced before all the old examples should be modified into new API or deleted in v2.0.

@chensuyue  who will do this delete action?

**Comment by zehao-intel** [comment]
Extension: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3671/

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3704/artifact/report.html)  
Please check int8/fp32 , new/last(weekly) performance gap.

**Comment by zehao-intel** [comment]
Performance drop is caused by the lack of warmup. I ignored it previously, the old API set it to 5.

Extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3712/

**Comment by chensuyue** [comment]
> Performance drop is caused by the lack of warmup. I ignored it previously, the old API set it to 5.
> 
> Extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3712/

please check accuracy issue.

**Comment by zehao-intel** [comment]
Rebuild extension: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3720/

**Comment by zehao-intel** [comment]
Extension: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3746/

File: examples/tensorflow/image_recognition/keras_models/inception_v3/quantization/ptq/run_benchmark.sh [hunk 3]
```diff
@@ -10,6 +10,9 @@ function main {
 
 # init params
 function init_params {
+  batch_size = 32
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/inception_v3/quantization/ptq/run_benchmark.sh:13]: fix the syntax

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3759/)

**Review by chensuyue — APPROVED** [review]
